2022/day9/Program.fs

```diff
@@ -0,0 +1,91 @@
+﻿// https://adventofcode.com/2022/day/9
+open System.IO
+
+type Pos = struct
+    val X: int
+    val Y: int
+    new (x: int, y: int) = { X = x; Y = y }
+end
+
+// Create a new position by moving from the previous position.
+let movePos (x: int) (y: int) (p: Pos) = Pos(p.X + x, p.Y + y)
+
+// Parse a move sequence into a sequence of move function calls.
+// This will convert multi-cell moves into a sequence 1-cell of moves.
+let parseMoveSeq (move: string) =
+    match move.ToUpper().Split(' ') with
+    | [| "U"; count |] -> seq { for c in 1 .. (int count) -> movePos 0 -1 }
```

Comment:
Seq computational expression (I call them list comprehesion because python) are perfectly fine, just want to give you another idea how you can write it so you can choose for yourself:
`    | [| "U"; count |] -> Seq.init (int count) (fun _ -> movePos 0 -1) `

I am not sure why you decided to structure it so you generate fuction calls that you later execute. My instincts tell me that just the data part is important here and you can act on it, but I think this is just personal preference.